docs/source/config.rst

```diff
@@ -0,0 +1,6 @@
+Configuration options
+=====================
+
+The configuration options specific to ``tomyst`` which you can use in your ``conf.py`` file are:
+
+#. **tomyst_static_file_path**: An array which takes path to your static folders, relative to the ``confdir``.
```

Comment:
thanks @AakashGfude this is really the `target` file path right?

sphinx will (by default) copy the `_static` folder to the root level of the target _build folder. This setting moves that location to match the source location (and isn't specifically used to fetch files)

Reply:
yes, that's true. You are right, I should be more verbose and clear about it.